options/m_property.c

```diff
@@ -352,7 +352,7 @@ void m_properties_print_help_list(struct mp_log *log,
     mp_info(log, "\nTotal: %d properties\n", count);
 }
 
-int m_property_bool_ro(int action, void* arg, int var)
+int m_property_bool_ro(int action, void* arg, bool var)
```

Comment:
Should you also change the cast below to bool?

Reply:
Yes, that looks like another memory error :thinking:

Reply:
done